src/OpenLoco/src/Config.h

```diff
@@ -204,6 +213,8 @@ namespace OpenLoco::Config
         bool usePreferredOwnerFace;
         ObjectHeader preferredOwnerFace;
 
+        MusicSortType sortMusicBy = MusicSortType::original;
```

Comment:
just adding this here does nothing. you need to write the code to load and save it from config as well. this happens in `readNew()` and `writeNewConfig()` in Config.cpp